worker/Dockerfile.template

```diff
@@ -49,7 +50,9 @@ RUN mkdir -p /etc/qemu \
 WORKDIR /usr/app
 
 COPY config config
-COPY entry.sh .
+# hack to conditionally copy over the ssh key only if it exists
+COPY entry.sh id.pub* ./
+RUN cp id.pub /root/id_rsa.pub; exit 0
```

Comment:
We should replace this with an env var like `CORE_PUBLIC_KEY` that is set by the Makefile and written to `id_rsa.pub` during entry.

Reply:
now no longer necessary